plugins/woocommerce-blocks/tests/e2e/tests/product-gallery/inner-blocks/product-gallery-large-image/product-gallery-large-image.block_theme.spec.ts

```diff
@@ -99,7 +99,8 @@ test.describe( `${ blockData.name }`, () => {
 					( el ) => el.style
 				);
 
-				expect( style.transform ).toBe( 'scale(1)' );
+				expect( style.transform ).not.toBe( 'scale(1.3)' );
+				expect( style.transform ).toBe( '' );
```

Comment:
The second expect "includes" the first one

```suggestion
				expect( style.transform ).toBe( '' );
```